web-marketplace/src/components/organisms/SellOrdersActionsBar/SellOrdersActionsBar.CreatePostButton.tsx

```diff
@@ -0,0 +1,24 @@
+import { useLingui } from '@lingui/react';
+
+import OutlinedButton from 'web-components/src/components/buttons/OutlinedButton';
+
+import { CREATE_POST } from './SellOrdersActionsBar.constants';
+
+export function SellOrdersActionsBarCreatePostButton({
+  onClickCreatePost,
+  isCreatePostButtonDisabled,
+}: {
+  onClickCreatePost: () => void;
+  isCreatePostButtonDisabled: boolean;
+}) {
+  const { _ } = useLingui();
+  return (
+    <OutlinedButton
+      onClick={onClickCreatePost}
+      className="mr-20"
+      disabled={isCreatePostButtonDisabled}
+    >
+      {_(CREATE_POST)}
```

Comment:
best to use `Trans` component in this case, check out https://lingui.dev/tutorials/react-patterns (without `msg`)

Reply:
oh yes, I knew about <Trans>, sorry I didn't use it.
Also sorry for the merge, I clicked the Github Merge button...

Reply:
that's ok, I believe the version with `msg` could work also since the content is just a simple string without sub-components but just trying to follow best practices

if you want to merge from github, make sure to open up the dropdown and select the second option "update with rebase", thanks

![image](https://github.com/user-attachments/assets/7234e798-4794-484a-bf7c-d33ac24fa4f0)

Reply:
@blushi I've reviewed this and I did use`{_(CREATE_POST)}` based on what I understand is the recommended way as per https://lingui.dev/tutorials/react-patterns#picking-a-message-based-on-a-variable.

Reply:
ok good to merge after conflicts resolution